cli/device.cpp

```diff
@@ -250,6 +252,12 @@ bool VulkanDevice::init_device(const Options &opts)
 	else
 		gpu_features.features.robustBufferAccess = VK_FALSE;
 
+	pipeline_stats = stats_feature.pipelineExecutableInfo;
```

Comment:
Extension is not checked for. The extension should not be enabled when want_pipeline_stats is false, similar to VK_AMD_shader_info. At least that extension would force-disable on-disk pipeline cache if the extension was just enabled, which is a serious performance concern when replaying.